src/index.ts

```diff
@@ -57,11 +57,8 @@ async function run() {
     const latestRelease = await octokit.request(`GET /repos/${srcRepo}/releases/latest`)
 
     const version = latestRelease.data.tag_name
-    const releaseId = latestRelease.data.id
 
-    const assets = await octokit.request(`GET /repos/${srcRepo}/releases/${releaseId}/assets`)
-
-    const tarballUrl = find(assets.data, a => a.name.includes("macos")).browser_download_url
```

Comment:
find is now unused as far as I can tell. Import needs to be removed.

Reply:
Good catch, updated!